Kernel/Syscalls/mmap.cpp

```diff
@@ -493,9 +493,6 @@ int Process::sys$munmap(void* addr, size_t size)
     // Try again while checkin multiple regions at a time
     // slow: without caching
     const auto& regions = space().find_regions_intersecting(range_to_unmap);
-    // if there still no regions found error out
-    if (!regions.size())
-        return -EINVAL;
```

Comment:
Squash this into the previous commit (43fefc6d76c56a0965ba90b237d8513547860d23), which adds these lines.

(Or leave it out completely.)

Reply:
Hmm, Github says my comment is "Outdated", but I'm pretty sure I commented on the right commit. If I just got confused, please ignore my comment.